playbook/app/pb_kits/playbook/pb_checkbox/_checkbox.scss

```diff
@@ -52,13 +52,19 @@ $transition: $transition_cubic;
     position: relative;
     width: 0;
     left: $offscreen;
+
     &:focus ~ .pb_checkbox_checkmark {
-      border-color: $primary_action;
+        box-shadow: 0px 0px 0px 2px $white, 0px 0px 0px 4px $primary;
```

Comment:
When we get a couple of these established, we should add a story for using the same mixin. There are going to be a ton that are very similar, if not exactly the same.